vignettes/customization.Rmd

```diff
@@ -0,0 +1,222 @@
+---
+title: "Customize your pkgdown website"
+output: rmarkdown::html_vignette
+description: >
+  Learn how to change the look and feel of pkgdown websites.
+vignette: >
+  %\VignetteIndexEntry{customization}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(pkgdown)
+```
+
+You can alter the design/styling of your pkgdown website to suit your brand and needs. 
+_Theming_ refers to being able to tweak colors, fonts, spacing etc. by using a Bootswatch template, Bootstrap variables, Bootstrap rules, or any custom CSS. 
+_Layout_ refers to being able to change the content of say the navbar, authors page, `<head>` tag by using pkgdown configuration or pkgdown templates.
+
+Another key concept related to styling your pkgdown website is whether you are tweaking _one_ package website, or several ones. 
+In the latter case you might want to share your style files in the form of a package.
+
+In this vignette we shall explain you the scope and tools for _theming_ and tweaking the _layout_ of pkgdown websites, and then explain how to best distribute your style when applied to more than one pkgdown website.
+
+## Pre-requisite: update to Bootstrap 4!
+
+Note that most theming features work only with Bootstrap 4, and that current development of pkgdown templates only happen for Bootstrap 4 templates. 
+If you are reading this vignette, please take a moment to update your website to Boostrap 4.
+
+```yaml
+template:
+  bootstrap: 4
+```
+
+It should not break your website, it should however change its styling a bit, for the better we hope.
```

Comment:
```suggestion
This has low risk of breaking your site, but you should preview and double-check just to be sure.
Overall, the site should look pretty similar, but there will be a few minor improvements.
```

We probably should have a bulleted list of the biggest changes in the NEWS file so that we can also mention in the blog post. IMO the most important is the font size.